keadm/cmd/keadm/app/cmd/edge/join_others.go

```diff
@@ -272,84 +267,3 @@ func runEdgeCore(withMqtt bool) error {
 	klog.Infoln(tip)
 	return nil
 }
-
-func createV1alpha1EdgeConfigFiles(opt *common.JoinOptions) error {
```

Comment:
Is there another one in join_windows?

Reply:
> Is there another one in join_windows?

No, because edgecore supports windows since v1.15, it does not involve config of v1alpha1.